terraform/moe_user_write.tf

```diff
@@ -0,0 +1,39 @@
+resource "aws_iam_user" "create_moe_user_write" {
+  name = "moe_user_write"
+  path = "/system/"
+}
+
+resource "aws_iam_access_key" "create_moe_user_write" {
+  user = "${aws_iam_user.create_moe_user_write.name}"
+}
+
+resource "aws_iam_user_policy" "create_moe_user_write_role" {
+  name = "moe_bucket_write_access"
+  user = "${aws_iam_user.create_moe_user_write.name}"
+
+  policy = <<EOF
```

Comment:
See other examples of creating a policy.  We create the policies separately instead of inline.